### PR TITLE
Update index.php

### PR DESCRIPTION
**Pull request by juliegund**
Julie is learning

### CODE REVIEW

File: www/index.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
 This page is hosted by your dampmam container! How awesome is that!
 
 <?
-echo "This is some PHP code! 2+2=".(2+2);
+echo "This is some PHP code! 2+2=".(2+5);
```
Comment (jpetazzo) [inline www/index.php:8]: This math thing is entirely new to me, let me call my plumber and we'll figure it out by filling 4g buckets with 6g buckets

Comment (jpetazzo) [inline www/index.php:8]: This math thing is entirely new to me, let me call my plumber and we'll figure it out by filling 4g buckets with 6g buckets

**Comment by soulshake** [comment]
\o/ :tada:

**Review by juliegund — COMMENTED** [review]
Jerome I can't find where to start again :)

**Pull request merged**